### DISCUSSION TITLE
I dont see the the github sponsors option on my personal account.

### DISCUSSION THREAD

**Discussion by raiyanyahya** [community/community#33587] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi Team,

I wanted to enable GitHub Sponsors on my personal github account but I dont see that option. Could someone please help ? HaveI accidently disabled it ? Does it exist for personal accounts or only for Organisations ?

/thanks

**Comment by cheshire137** [comment, marked as answer]
Hi there!

> HaveI accidently disabled it ?

It looks like you disabled your Sponsors profile back in May. I kicked it back to an editable state for you, you should be able to access https://github.com/sponsors/raiyanyahya/dashboard now.

**Reply by raiyanyahya** [reply]
wow! Thank you for the quick help.
